src/app/app-routing.module.ts

```diff
@@ -4,13 +4,13 @@ import { UserlistComponent } from './userlist/userlist.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
 import { CaseComponent } from './case/case.component';
 import { AssignedCasesComponent } from './AssignedCases/AssignedCases.component';
-
+import { config } from 'rxjs';
 
 const routes: Routes = [
   { path: '', redirectTo: '/dashboard', pathMatch: 'full' },
   { path: 'dashboard', component: DashboardComponent },
   { path: 'userlist', component: UserlistComponent },
-  { path: 'case', component: CaseComponent },
+  { path: 'positive-case', component: CaseComponent },
```

Comment:
Do we want to change the case component to the PositiveCaseComponent? I think that name is more descriptive of what it is.